BigBaseV2/src/services/gta_data/gta_data_service.cpp

```diff
@@ -135,81 +129,453 @@ namespace big
+					{
+						pugi::xpath_node_set items = doc.select_nodes("/CVehicleModelInfo__InitDataList/InitDatas/Item");
+						for (pugi::xpath_node item_node : items)
+						{
+							pugi::xml_node item = item_node.node();
+
+							nlohmann::json veh;
+
+							veh["Name"] = item.child("modelName").text().as_string();
+							veh["HandlingId"] = item.child("handlingId").text().as_string();
+
+							rage::joaat_t model_hash = rage::joaat(veh["Name"]);
+
+							if (veh_hashes.find(model_hash) != veh_hashes.end())
+							{
+								continue;
+							}
+							veh_hashes[model_hash] = {};
+
+							veh["Hash"] = model_hash;
+							veh["SignedHash"] = (int32_t)model_hash;
+							std::stringstream s;
+							s << HEX_TO_UPPER(model_hash);
+							veh["HexHash"] = s.str();
+
+							veh["DlcName"] = "TODO"; // todo: how DurtyFree generate it?
+
+							veh["HandlingId"] = item.child("handlingId").text().as_string();
+
+							veh["LayoutId"] = item.child("layout").text().as_string();
+
+							const std::string manufacturer = item.child("vehicleMakeName").text().as_string();
+							if (manufacturer.size())
+							{
+								veh["Manufacturer"] = manufacturer;
+								veh["ManufacturerHash"] = rage::joaat(manufacturer);
+								veh["ManufacturerDisplayName"] = nullptr;
+							}
+							else
+							{
+								veh["Manufacturer"] = "";
+								veh["ManufacturerHash"] = -1;
+								veh["ManufacturerDisplayName"] = nullptr;
+							}
+
+							const auto vehicle_class = std::string(item.child("vehicleClass").text().as_string());
+							if (vehicle_class.size() > 3)
+								veh["Class"] = vehicle_class.substr(3);
+							else
+								veh["Class"] = nullptr;
+
+							veh["ClassId"] = 0; // todo: how DurtyFree generate it?
+
+							const auto vehicle_type = std::string(item.child("type").text().as_string());
+							if (vehicle_type.size() > 13)
+								veh["Type"] = vehicle_type.substr(13);
+							else
+								veh["Type"] = nullptr;
+
+							const auto plate_type = std::string(item.child("plateType").text().as_string());
+							if (plate_type.size() > 4)
+								veh["PlateType"] = plate_type.substr(4);
+							else
+								veh["PlateType"] = nullptr;
+
+							const auto dashboard_type = std::string(item.child("dashboardType").text().as_string());
+							if (dashboard_type.size() > 4)
+								veh["DashboardType"] = dashboard_type.substr(4);
+							else
+								veh["DashboardType"] = nullptr;
+
+							const auto wheel_type = std::string(item.child("wheelType").text().as_string());
+							if (wheel_type.size() > 4)
+								veh["WheelType"] = wheel_type.substr(4);
+							else
+								veh["WheelType"] = nullptr;
+
+							const std::string flags_str = item.child("flags").text().as_string();
+							veh["Flags"] = split_str_into_json_array(flags_str, " ");
+
+							veh["Seats"] = VEHICLE::GET_VEHICLE_MODEL_NUMBER_OF_SEATS(model_hash);
+
+							veh["Price"] = -1; // todo: how DurtyFree generate it?
```

Comment:
CHandlingData has this information I believe.

https://github.com/Yimura/GTAV-Classes/blob/1da236c3c6ca5586dc9e1069dde1fbfd49641882/vehicle/CHandlingData.hpp#L74